<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentSchedule</class>
 <widget class="QWidget" name="PaymentSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>13</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>График платежей</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="payment_schedule_display">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>461</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>53</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>53</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ставка, %:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>53</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок (в месяцах):</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>65</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="schedule_sum_display">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>83</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="schedule_rate_display">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>83</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="schedule_term_display">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>83</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>100</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>580</y>
     <width>551</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="schedule_total_payment_display">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>598</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="schedule_overpayment_display">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>598</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="schedule_total_payment">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>568</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string> Общая выплата:</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>615</y>
     <width>551</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="schedule_overpayment">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>568</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Переплата:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
